src/test/resources/template/ParameterReuseRecipe$1_before.java

```diff
@@ -16,7 +16,19 @@
 package foo;
 import org.openrewrite.java.*;
 
+/**
+ * OpenRewrite `before` template created for  `foo.ParameterReuseRecipe$1`.
+ */
 public class ParameterReuseRecipe$1_before {
+    /**
+     * Instantiates a new {@link ParameterReuseRecipe$1_before} instance.
```

Comment:
There's also a doclint check that validates `@link`s; I'm not sure, but it could be that the `$` should be a `.` or `#`.

Reply:
Good to know! Removed the `@link`; life's too short.